### ISSUE TITLE
Suggestion: Isolate word frequency into separate file

### ISSUE THREAD

**Issue by isConic** [David-Hinschberger/Monoalphabetic-Cipher-Decipher#1]
The file will be re-usable and swappable as a result.

**Comment by David-Hinschberger** [comment]
Do you think it should be a module or just a text file that gets read in?

**Comment by isConic** [comment]
I've created function in PR #2 that loads it in. 
For now it's in a separate JSON file.

**Comment by David-Hinschberger** [comment]
Why JSON that requires parsing when it can be done with a separate Python module? 

For example, in a file ratings.py,

`trigrams = {'THE':0.01814036094632001,'AND':0.007252281087500791}`

and it'd just be imported as `import trigrams from ratings`

**Comment by isConic** [comment]
Either are viable. 

### Case for a static representation of data . 
If someone wants to re-use your trigrams (lets say a `rust` version of this is written), the JSON is language agnostic and therefore more portable.     
  
Likewise lets say the `rust` version is already written and it uses a more accurate trigram JSON file.  The process of acquiring and reusing their trigrams to improve your model will not require explicit knowledge of `rust`.  
  
Static stores of data are re-usable. Language specific interfaces to popular data formats exist so it's easier to get to no matter what the language.   

**tldr:** in terms of effort reusability. language specific encoding of the trigrams is `O(n)`
data format specific encodings are `O(1)`. 

### Case for a module

Modules are easier to use/re-use if the user is familiar with python. Modules can be used as a layer between the user and the data and provide added functionality/mechanisms to handle things like: 

- fetching large datasets responsibly 
- supporting lazy loading 
- interface for storage or data-versioning .   
- caching mechanisms to hedge IO performance cost 

A good example of where a dataset module works out would be `tensorflow`s example datasets. Tensorflow has a built in module to handle that with built in support for lazy loading/evaluation. 

https://www.tensorflow.org/tutorials/load_data/images . 

**tldr:** modules are a mature way to handle access and manipulation of the data.    
  
### This case

The dictionary was isolated into a separate file because it was causing my editor grief (Pycharm Pro and Sublime Text)   
  
`main.py` was difficult to navigate. without scrolling over a few dozen line-wrapped trigram values.  The simplest remedy I could think of was moving them into a JSON file.  
  
I think creating a data-access module is definitely a good idea but may have to happen after more refactoring.

**Comment by David-Hinschberger** [comment]
Alright I'll go on with my idea of the module.
